convoy.json.example

```diff
@@ -0,0 +1,80 @@
+{
+  "database": {
+    "environment": "dev",
+    "dsn": "mongodb://root:rootpassword@localhost:27017/convoy?authSource=admin"
```

Comment:
The connection will fail in the docker network because `localhost` will not resolve to the actual `mongodb` container 
```suggestion
    "dsn": "mongodb://root:rootpassword@mongodb:27017/convoy?authSource=admin"
```

Reply:
Exactly, why I removed it. Formerly, as well we had `convoy-docker.json` for running convoy in a container. 

I suggest we just rename it to `<insert-mongodb-dsn>` same for redis. @Dotunj